src/main/java/com/agency04/sbss/pizza/model/Size.java

```diff
@@ -0,0 +1,25 @@
+package com.agency04.sbss.pizza.model;
+
+/**
+ *  Sizes of pizza
+ *  {@link #SMALL}
+ *  {@link #MEDIUM}
+ *  {@link #LARGE}
+ */
+public enum Size {
```

Comment:
Promijeniti naziv enuma tako da na prvu bude jasno na što se odnosi. Također pripaziti na to da ako ostali enumi imaju prefiks "E" da i ovaj to ima.